docs/tutorials/analysis/1D/spectral_analysis_hli.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>Why is the copy needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/4044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>removed</p>